readme.md

```diff
@@ -1,692 +1,67 @@
-# Gataca-QR
+# Gataca-QR Components
 
-This component allows an easy integration to display a GATACA QR code.
-You only need to install on a front component to scan presentation requests from the [GATACA Wallet](https://gataca.io/products/wallet) app.
+Since V2, this library is now a list of components provided by Gataca to help the integration of its backends components into your front services.
 
-## Installing the component
+To use this components (except for the QR Display), you will need an instance of any of the Gataca Components (Connect and Certify). You can also use it with onPremise instances over Gataca Studio.
 
-### Script tag
-- Put a script tag similar to this `<script src='https://unpkg.com/gatacaqr@1.4.2/dist/gatacaqr.js'></script>` in the head of your index.html
-- Then you can use the element anywhere in your template, JSX, html etc
+## Component List
 
-### Node modules
-- Run `npm install gatacaqr --save`
-- Put a script tag similar to this `<script src='node_modules/gatacaqr/dist/gatacaqr.js'></script>` in the head of your index.html
-- Then you can use the element anywhere in your template, JSX, html etc
+### [Gataca QRDisplay](src/components/gataca-qrdisplay/readme.md)
 
-## Styles & Personalization
+Just to display QRs with the Gataca format in any required component
 
-It allows to integrate 2 slots, named "title" and "description", to provide further integration to the user upon display of the QR.
-Both should be passed to the component as params: 
-- qr-modal-title
-- qr-modal-description
+### [Gataca QR](src/components/gataca-qr/readme.md)
 
-In order to control how the QR code is previsualized the ideal situation is to encapsulate the QR component in a tag with the styles on your control. The component will adapt and fill the container.
+Main component of the library. It has been reworked and simplified, while keeping all it's flexibility.
 
-````html
-<div class="your-container-styles">
+It displays a qr as an overflow panel to integrate into your site. It requires that you have a backend service integrated with Gataca Connect or Certify.
```

Comment:
Studio instead of Connect / Certify

Reply:
Now it's not needed because we must change the documentation before

Reply:
The thing is this works also for onpremise clients :)